index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <script src="script.js" defer></script>
+    <title>Welcome to Pro-To-Type</title>
+
+</head>
+
+<body>
+
+    <main>
+
+        <h1>Pro-To-Type</h1>
+        <div id="rulesContainer">
+            <h2 id="rulesSubheading">Rules</h2>
+            <p id="rulesText">Test your typing speed below! Once you start typing the timer begins. Be careful, because
+                once you press space you can no longer edit the word, and you must move on to the next word. The word
+                you are currently typing will be yellow, words spelled correctly will turn green, incorrect words will
+                turn red. It'll take a Pro-To-Type this correctly!!!</p>
+        </div>
+        <div class="buttonContainer1">
```

Comment:
buttonContainer1 is not a very helpful name - can you chage it to something more explicit?